### PR TITLE
Nhost login + Navbar on Desktop

### PR DESCRIPTION
**Pull request by rafael-longeville**

### CODE REVIEW

**Comment by juicygoose** [comment]
nice ! you now have themisto app and scratch in the repo , could you clean one of the two (themisto) as it will not be used?
thanks !

File: apps/scratch/pages/_app.tsx [hunk 1]
```diff
@@ -1,6 +1,47 @@
...
-import type { AppProps } from 'next/app'
+import type { AppProps } from "next/app";
+import { NextApiResponse } from "next";
+import { NhostProvider } from "@nhost/nextjs";
+import { nhost } from "libs";
+import { useAuthenticationStatus } from "@nhost/nextjs";
+
+// --- Images --- //
+import logo from "../public/yalink-black-letters.svg";
+import { ChakraProvider, extendTheme } from "@chakra-ui/react";
```
Comment (juicygoose) [inline apps/scratch/pages/_app.tsx:9]: I think we should now go with one provider only to keep things simple. I would favor mantine over chakra for its amount of components and all the added value of the mantine forms. what do you think?

File: apps/scratch/pages/_app.tsx [hunk 2]
```diff
@@ -1,6 +1,47 @@
...
+import { Flex, Box } from "@chakra-ui/react";
+import { MantineProvider } from "@mantine/core";
+import { Sidebar } from "mantine-core";
+const theme = extendTheme({
+  fonts: {
+    heading: "Source Sans Pro",
+  },
+  colors: {
+    yalink: {
```
Comment (juicygoose) [inline apps/scratch/pages/_app.tsx:18]: in mantine for example, the themes are more robust to light / dark mode compared to this

File: apps/scratch/pages/_app.tsx [hunk 3]
```diff
@@ -1,6 +1,47 @@
...
+      green: "#003B36",
+      yellow: "#FFCD00",
+      teal: "#0F665D",
+    },
+  },
+});
 
 export default function App({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+  const refreshToken = nhost.auth.getSession()?.refreshToken;
```
Comment (juicygoose) [inline apps/scratch/pages/_app.tsx:29]: this shouldn't be outside of the nhost provider and directly put into the navbar component

File: apps/scratch/pages/login/index.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import Head from "next/head";
+import { Login } from "ui";
+
+export default function LoginPage() {
+  return (
+    <>
+      <Head>
+        <title>Create Next App</title>
```
Comment (juicygoose) [inline apps/scratch/pages/login/index.tsx:8]: To be changed with yalink app and the details of our app

File: package.json [hunk 5]
```diff
@@ -26,8 +26,7 @@
     "node": ">=14.0.0"
   },
   "dependencies": {
-    "axios": "^1.2.1",
-    "webpack": "^5.75.0"
+    "axios": "^1.2.1"
```
Comment (juicygoose) [inline package.json:29]: I don't even think we need axios here ?

File: packages/mantine-core/Sidebar/GroupedLinks.tsx [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+  TablerIcon,
+  IconCalendarStats,
+  IconChevronLeft,
+  IconChevronRight,
+} from "@tabler/icons";
+import { useRouter } from "next/router";
+import { Box as ChakraBox } from "@chakra-ui/react";
+import { pushToRoute } from "./lib";
+
+const useStyles = createStyles((theme) => ({
```
Comment (juicygoose) [inline packages/mantine-core/Sidebar/GroupedLinks.tsx:21]: useStyles can co in a styles.ts file next to the component

File: packages/mantine-core/Sidebar/Sidebar.tsx [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+import { createStyles, Navbar, Group } from "@mantine/core";
+import {
+  IconHomeQuestion,
+  IconLogout,
+  IconQuestionCircle,
+  IconSettings,
+} from "@tabler/icons";
+import GroupedLinks from "./GroupedLinks";
+
+const useStyles = createStyles((theme, _params, getRef) => {
```
Comment (juicygoose) [inline packages/mantine-core/Sidebar/Sidebar.tsx:24]: same here

File: packages/ui/Login/Inputs/Password.tsx [hunk 8]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
```
Comment (juicygoose) [inline packages/ui/Login/Inputs/Password.tsx:1]: either we import this everywhere or we remove it everywhere maybe

File: packages/ui/Login/Login.tsx [hunk 9]
```diff
@@ -1,68 +1,62 @@
...
+
   const formik = useFormik({
     initialValues: {
-      email: '',
-      password: '',
+      email: "",
+      password: "",
     },
-    // validate: values => {
+    // validate: (values) => {
```
Comment (juicygoose) [inline packages/ui/Login/Login.tsx:42]: values should probably be validated

File: packages/ui/Login/linksGroup.tsx [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+  createStyles,
+} from "@mantine/core";
+import {
+  TablerIcon,
+  IconCalendarStats,
+  IconChevronLeft,
+  IconChevronRight,
+} from "@tabler/icons";
+
+const useStyles = createStyles((theme) => ({
```
Comment (juicygoose) [inline packages/ui/Login/linksGroup.tsx:18]: same for usestyle here

File: apps/scratch/pages/_app.tsx (hunk 1 above)
Comment (rafael-longeville) [reply apps/scratch/pages/_app.tsx:9]: Why not, it's some more work to migrate the chakra ui pages that we have to mantine but i think it has some added value to so let's do it

File: packages/ui/Login/linksGroup.tsx (hunk 10 above)
Comment (rafael-longeville) [reply packages/ui/Login/linksGroup.tsx]: This file has been deleted

**Pull request merged**